aries-site/src/examples/components/button/ButtonAlignmentTable.js

```diff
@@ -0,0 +1,126 @@
+import React, { Fragment } from 'react';
+
+import {
+  Anchor,
+  Box,
+  Card,
+  Table,
+  TableCell,
+  TableHeader,
+  TableBody,
+  TableRow,
+  Text,
+} from 'grommet';
+
+const data = [
+  {
+    id: 1,
+    alignment: 'Left-aligned',
+    useCases: [
+      {
+        label: 'Side Drawers',
+        href: '/components/layer#side-drawer-modal',
+      },
+      {
+        label: 'Forms',
+        href: '/templates/forms',
+      },
+      {
+        label: 'Filters',
+        href: '/templates/filtering',
+      },
+    ],
+  },
+  {
+    id: 2,
+    alignment: 'Right-aligned',
+    useCases: [
+      {
+        label: 'Wizards',
+        href: '/templates/wizard',
+      },
+      {
+        label: 'Confirmation Modals',
+        href: '/components/fileinput#fileinput-with-file-removal-confirmation',
+      },
+      {
+        label: 'Dialogs',
+      },
+      {
+        label: 'Notifications',
+      },
+    ],
+  },
+];
+
+const columns = [
+  {
+    property: 'alignment',
+    label: 'Alignment',
+  },
+  {
+    property: 'useCases',
+    label: 'Use Cases',
+  },
+];
+
+export const ButtonAlignmentTable = () => (
+  <Card alignSelf="start" elevation="none" pad="medium" width="large">
+    {/* style tag is necessary in order for table not 
+    to exceed container width */}
+    <Table style={{ tableLayout: 'fixed', width: '100%' }}>
+      <TableHeader>
+        <TableRow>
+          {columns.map(c => (
+            <TableCell
+              border="bottom"
```

Comment:
It feels like we are being inconsistent with our table styling. This styling differs from how they are applied on [Voice & Tone](https://design-system.hpe.design/foundation/voice-and-tone#other-common-problems-and-how-to-address-them) and our [DataTables](https://design-system.hpe.design/components/datatable).

I feel like we should be leaning on the theme here.

In addition, the [Figma file](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4153%3A9026) should probably also be using a Table component as opposed to text + lines to mimic a table, but isn't one...

There could be some rationale I am missing, but without that, I'm advocating for consistency and consuming the Design System's theme.

Reply:
I am good with that an just falling back on what we get out of the box from the theme.